src/components/Dashboard/TodoList.vue

```diff
@@ -181,64 +172,45 @@
                   <v-btn text loading disabled />
                 </template>
 
-                <!-- draft correction or alteration or conversion (staff only) -->
-                <template v-else-if="isStatusDraft(item)
-                  && (isTypeCorrection(item) || isTypeAlteration(item) || isTypeConversion(item))
-                  && isRoleStaff"
-                >
+                <!-- new annual report -->
```

Comment:
This new annual report section, and the new conversion section below, were simply moved up to be checked before the other draft states.